datarepo-clienttests/src/main/java/utils/KubernetesClientUtils.java

```diff
@@ -35,6 +35,8 @@
   public static final String componentLabel = "app.kubernetes.io/component";
   public static final String apiComponentLabel = "api";
 
+  private static String namespace;
```

Comment:
@marikomedlock  - I'm worried that this is now going to be confusing. As it stands, there are at least three different "namespaces" we reference in this file. 
1. For some of the methods, the namespace can still can get passed in as an argument.
2. The namespace is sometimes retrieved off the metadata of the deployment.
3. And in buildKubernetesClientObject, it's set from server.namespace. 

So, I'm not sure if we should do a minor overhaul and always reference this new namespace. Or, if should just name this something a little more specific than just "namespace."